src/librustdoc/passes/collect_intra_doc_links.rs

```diff
@@ -118,7 +196,9 @@ impl<'a, 'tcx> LinkCollector<'a, 'tcx> {
                     .flat_map(|imp| cx.tcx.associated_items(*imp).in_definition_order())
```

Comment:
This is buggy: it should use `find_by_name_and_namespace` to avoid false positives.

Reply:
I think fixing this should go in a follow-up PR.